packages/web-client/app/components/card-pay/deposit-workflow/transaction-amount/index.hbs

```diff
@@ -53,14 +53,24 @@
         </CardPay::LabeledValue>
       {{/unless}}
     </div>
-    {{#if this.errorMessage}}
-      <CardPay::ErrorMessage as |supportURL|>
-        {{this.errorMessage}} Please try again if you want to continue with this workflow, or contact <a href={{supportURL}} target="_blank" rel="noopener noreferrer">Cardstack support</a>.
-      </CardPay::ErrorMessage>
-    {{/if}}
     <div class="transaction-amount__footnote">
       * The actual value depends on the current exchange rate and is determined at the time of authorization.
     </div>
+    {{#if (and this.errorMessage this.depositTaskRetriable)}}
```

Comment:
Although there is no authoritative correct spelling of retriable/retryable, I would go with the "y" version. Background reading: https://english.stackexchange.com/questions/305273/retriable-or-retryable